From 17d92d96f74a2743226b8ddd86b9f54a0b56d3ce Mon Sep 17 00:00:00 2001
From: Kumar Appaiah <a.kumar@alumni.iitm.ac.in>
Date: Wed, 8 Aug 2012 15:29:10 -0500
Subject: [PATCH 1/2] Comment the Javascript code

In order for the code to be valid XHTML 1.1, inline Javascript has to
be commented out in HTML perspective. This commit introduces that
comment in library.php.
---
 library.php |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/library.php b/library.php
index 0414568..25c2ef5 100644
--- a/library.php
+++ b/library.php
@@ -1230,6 +1230,7 @@ function html_header($subtitle="Buildd information pages", $js=false, $raw=false
 
   echo "
 <script type=\"text/javascript\">
+<!--
 $(document).ready(function () {
   $(\"#pkg_field\").focus();
 
@@ -1245,6 +1246,7 @@ $(document).ready(function () {
     }
   });
 });
+//-->
 </script>
 ";
 
-- 
1.7.10.4

